test/tests/val-2a/tests

```diff
@@ -0,0 +1,18 @@
+[Tests]
+  design = 'EquilibriumBC.md'
+  issues = '#12 #188'
+  validation = 'val-2a.md'
+  [val-2a_TMAP4_csvdiff]
+    type = CSVDiff
+    input = 'val-2a_base.i val-2a_TMAP4.i'
+    csvdiff = val-2a_TMAP4_out.csv
+    heavy = true
+    requirement = 'The system shall be able to model Ion implantation based on the requirements and conditions listed in the TMAP4 V&V report'
+  []
```

Comment:
```suggestion
  []
  [val-2a_TMAP4_exodiff_heavy]
    type = exodiff
    input = 'val-2a_base.i val-2a_TMAP4.i'
    exodiff = val-2a_TMAP4_out.e
    prereq = val-2a_TMAP4_csvdiff_heavy
    should_execute = false # this test relies on the output files from val-2a_TMAP4_csvdiff_heavy, so it shouldn't be run twice
    heavy = true
    requirement = 'The system shall be able to model Ion implantation based on the requirements and conditions listed in the TMAP4 V&V report and generate an exodus file'
  []
  [val-2a_TMAP4_csvdiff]
    type = CSVDiff
    input = 'val-2a_base.i val-2a_TMAP4.i'
    cli_args = ''
    csvdiff = val-2a_TMAP4_out.csv
    requirement = 'The system shall be able to model Ion implantation based on the requirements and conditions listed in the TMAP4 V&V report'
  []
  [val-2a_TMAP4_exodiff]
    type = exodiff
    input = 'val-2a_base.i val-2a_TMAP4.i'
    exodiff = val-2a_TMAP4_out.e
    prereq = val-2a_TMAP4_csvdiff
    should_execute = false # this test relies on the output files from val-2a_TMAP4_csvdiff, so it shouldn't be run twice
    requirement = 'The system shall be able to model Ion implantation based on the requirements and conditions listed in the TMAP4 V&V report and generate an exodus file'
  []
```
Let's also test the exodus file (you'll need to add the gold), and create non-heavy versions of the test to track things that would affect this case in regular tests (update the cli_args to reduce the computational costs - maybe coarsen the mesh, maybe use larger time steps, maybe reduce the simulated end time, maybe slightly loosen tolerances - maybe do a bit of everything). The non-heavy tests are not meant to be very accurate, but they should capture the important physics well enough that a change in the code used here could be flagged by the tests. Also make sure to update the requirements (look at other tests for inspiration).